docs/content/interacting/consistency.mdx

```diff
@@ -55,14 +55,16 @@ if (date_modified + cache_time_to_live_period > Date.now()) {
+| check-query-cache-enabled        | Enables in-memory caching of Check subproblems. For example, if you have a relation `define viewer: owner or editor`, and the query is `Check(user:anne, viewer, doc:1)`, we'll evaluate the `owner` relation and the `editor` relation and cache both results: `(user:anne, viewer, doc:1) -> allowed=true` and `(user:anne, owner, doc:1) -> allowed=true`.     |
+| check-query-cache-ttl            | Specifies the time that items will be kept in the cache of Check subproblems                                                                                                                                                                                                                                                                                      |
+| check-iterator-cache-enabled     | Enables in-memory caching of database iterators. Each iterator is the result of a database query, for example, usersets related to a specific object, or objects related to a specific user, up to a certain number of tuples per iterator                                                                                                                        |
+| check-iterator-cache-max-results | Configures the number of tuples that will be stored for each database iterator                                                                                                                                                                                                                                                                                    |
+| check-iterator-cache-ttl         | Specifies the time that items will be kept in the cache of database iterators                                                                                                                                                                                                                                                                                     |
+| cache-controller-enabled         | When enabled, cache controller will verify whether check subproblem cache and check iterator cache needs to be invalidated when there is a check or list objects API request. The invalidation determination is based on whether there are recent write or deletes for the store. This feature allows a larger check-query-cache-ttl and check-iterator-cache-ttl.|
+| cache-controller-ttl             | Specifies the time that cache controller result is cached. While cache controller result is cached, server will not read datastore to check whether subproblem cache and iterator cache needs to be invalidated.                                                                                                                                                  |
```

Comment:
"Specifies the time that **the** cache controller result is cached. While **the** cache controller result is cached, **the** server will not read **the** datastore to check whether subproblem cache and iterator cache needs to be invalidated."

Reply:
"Specifies the time that the cache controller result is cached" does not make sense, and it leaks an implementation detail (that the cache controller internally _also_ uses the cache)

Can you change it to 

> Specifies how frequently the cache controller checks for Writes occurring